Samples.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29806.167
+# Visual Studio Version 17

```

Comment:
This doesn't really need Visual Studio 17.x, even though the new project requires BuildInsights SDK for msvc 17.10 Preview 3 or later. For now, I can revert this.